requirements.txt

```
@@ -0,0 +1,13 @@
+anyio==3.7.0
+certifi==2023.5.7
+exceptiongroup==1.1.2
+h11==0.14.0
+httpcore==0.17.3
+httpx==0.24.1
+idna==3.4
+Pillow==10.0.0
+pypng==0.20220715.0
+python-telegram-bot==20.3
+qrcode==7.4.2
+sniffio==1.3.0
+typing_extensions==4.7.1
```

Only 3 should be put here

```suggestion
Pillow==10.0.0
python-telegram-bot==13.12
qrcode==7.4.2
```